special-pages/pages/new-tab/app/privacy-stats/components/PrivacyStats.js

```diff
@@ -94,58 +96,13 @@ function PrivacyStatsConfigured({ parentRef, expansion, data, toggle }) {
     );
 }
 
-/**
- * @param {object} props
- * @param {Expansion} props.expansion
- * @param {number} props.recent
- * @param {boolean} props.canExpand
- * @param {() => void} props.onToggle
- * @param {import("preact").ComponentProps<'button'>} [props.buttonAttrs]
- */
-export function Heading({ expansion, canExpand, recent, onToggle, buttonAttrs = {} }) {
-    const { t } = useTypedTranslationWith(/** @type {enStrings} */ ({}));
-    const [formatter] = useState(() => new Intl.NumberFormat());
-
-    const none = recent === 0;
-    const some = recent > 0;
-    const alltime = formatter.format(recent);
-    const alltimeTitle = recent === 1 ? t('stats_countBlockedSingular') : t('stats_countBlockedPlural', { count: alltime });
-
-    return (
-        <div className={styles.heading}>
-            <span className={styles.headingIcon}>
-                <img src="./icons/shield.svg" alt="Privacy Shield" />
-            </span>
-            {none && <h2 className={styles.title}>{t('stats_noRecent')}</h2>}
-            {some && <h2 className={styles.title}>{alltimeTitle}</h2>}
-            {recent > 0 && canExpand && (
-                <span className={styles.widgetExpander}>
-                    <ShowHideButton
-                        buttonAttrs={{
-                            ...buttonAttrs,
-                            'aria-expanded': expansion === 'expanded',
-                            'aria-pressed': expansion === 'expanded',
-                        }}
-                        onClick={onToggle}
-                        text={expansion === 'expanded' ? t('stats_hideLabel') : t('stats_toggleLabel')}
-                        shape="round"
-                    />
-                </span>
-            )}
-            {recent === 0 && <p className={styles.subtitle}>{t('stats_noActivity')}</p>}
-            {recent > 0 && <p className={cn(styles.subtitle, styles.uppercase)}>{t('stats_feedCountBlockedPeriod')}</p>}
-        </div>
-    );
-}
-
```

Comment:
In this upcoming PR we have a second version of this heading, so just moving it to it's own component for easier comparison later.